DPoP for Okta resources (#4526)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/dpop/main/index.md

### Create the JSON Web Token

Create a DPoP proof [JWT](https://www.rfc-editor.org/rfc/rfc7519). A DPoP proof JWT includes a header and payload with claims, and then you sign the JWT with the private key from the [previous section](#create-a-json-web-key).

Use your internal instance to sign the JWT for a production org. See this [JWT generator](https://github.com/jwtk/njwt) for an example of how to make and use JWTs in Node.js apps. For testing purposes only, you can use this [JWT tool](https://jwt.io/) to build, sign, and decode JWTs. See [Use the JWT tool](#use-the-jwt-tool).

#### Parameters and claims

Include the following required parameters in the JWT header:

> * `typ`: Type header. Declares that the encoded object is a JWT and meant for use with DPoP. This example uses `dpop+jwt`.

Reviewer: ```suggestion
* `typ`: Type header. Declares that the encoded object is a JWT and meant for use with DPoP. This must be `dpop+jwt`.
```

        "kid": "XUl71vpgPXgxSTCYHbvbEHDrtj-adpVcxXH3TKjKe7w",
        "alg": "RS256",
        "n": "4LuWNeMa7.....zLvDWaJsF0"
      }
    }
  ```

3. In the **PAYLOAD** section, build the JWT payload and include the following claims:

  ```json
    {
      "htm": "POST",
      "htu": "http://${yourOktaDomain}/oauth2/default/v1/token",
      "iat": 1516239022
    }
  ```

4. In the **VERIFY SIGNATURE** section:

    * **First box:** Paste the public key (X.509 PEM format) from the previous section.
    * **Second box:** Paste the private key (X.509 PEM format).

5. Copy the JWT that appears in the **Encoded** section.

### Build the request

> Your next step is to build the POST request to the `/token` endpoint for an access token. Two requests to the `/token` endpoint are necessary. The initial request obtains the `dpop-nonce` header value from the authorization server. The second request includes an updated JWT with the `dpop-nonce` header value in the JWT payload.

Reviewer: nit:
```suggestion
Your next step is to build the POST request to the `/token` endpoint for an access token. Two requests to the `/token` endpoint are necessary. The initial request obtains the `dpop-nonce` header value from the authorization server. The second request includes an updated JWT with the `dpop-nonce` header value in the JWT payload. Once you have received a `nonce` value from the `/token` endpoint you may continue to use that value until you receive an error with a new `dpop-nonce` header.
```
